### ISSUE TITLE
Hyperparameters of Proxy_NCA

### ISSUE THREAD

**Issue by say2sarwar** [tjddus9597/Proxy-Anchor-CVPR2020#11]
Can you please list the parameters(LR, epochs, etc) for Proxy_NCA?

**Comment by sung-yeon-kim** [comment]
We used the same hyperparameters for Proxy-NCA as our training with loss, except for the scale parameter described in "code/losses.py".
